src/main.cpp

```diff
@@ -61,7 +61,23 @@ int main(int argc, char * argv[]) {
         args.printUsage();
         return 0;
     }
-
+    if(args.getSettingsPathSet()) {
+        QDir settingsPath(args.getSettingsPath());
+        qDebug() << "Settings Path Provided: " << settingsPath.absolutePath();
```

Comment:
OK, for writing in command line what i do, use stdout like printf, cout what I do, or the pop up dialog window is enough